src/utils.jl

```diff
@@ -0,0 +1,65 @@
+"""
+Efficiently reduce a BitArray with or (|)
+"""
+function or(B::BitArray)
```

Comment:
`all(B)` and `!all(B)`. Use the standard iterators from Base instead of renaming them

Reply:
It seems I need to sit down and read the base documentation end to end - these do exactly the same thing?

Reply:
yes.